src/earn/EarnConfirmationScreen.tsx

```diff
@@ -74,20 +78,50 @@ export default function EarnConfirmationScreen({ route }: Props) {
     [withdrawToken, pool.pricePerShare, inputAmount]
   )
 
-  // Will need this to handle preparing a claim transaction, a withdrawal transaction and a withdrawal and claim transaction
   const {
-    result: prepareTransactionsResult,
-    loading: isPreparingTransactions,
-    error: prepareTransactionError,
-  } = usePrepareWithdrawAndClaimTransactions({
-    amount: new BigNumber(withdrawAmountInDepositToken).dividedBy(pool.pricePerShare[0]).toString(),
-    pool,
-    walletAddress,
-    feeCurrencies,
-    hooksApiUrl,
-    rewardsPositions,
-    useMax,
-  })
+    // @ts-expect-error prepareTransactionsResult & swapTransaction are only set when mode === 'deposit' or 'swap-deposit'
```

Comment:
I don't understand these two lines. 
1. Seems like a red flag to do a ts-ignore here. if the return values of the different modes are that different then should they maybe be changed at the prepare transaction level? 
2. swapTransaction doesn't even seem to be used in the code.

Reply:
This @ts-expect-error is temporary; I'll remove it before moving the PR out of draft and resolving the type issues. While `swapTransaction` isn't currently used on this page there's [a bug](https://linear.app/valora/issue/ACT-1398) with the bottom sheet implementation that will likely require `swapTransaction` to be handled from a confirmation screen.

Reply:
This `// @ts-expect-error` is no longer present but the one on `EarnEnterAmount.tsx` is still present; I'll see how this can be removed.